Providers/Profile.swift

```diff
@@ -361,16 +351,16 @@ public class BrowserProfile: Profile {
         }
     }()
 
-    private var dbCreated = false
```

Comment:
I think a solution to this bug is to make `dbCreated` static! After all, if we have a singleton `BrowserDB` that's shared between `Profile` instances (which is wrong, of course), then the bug is that each `Profile` instance has its own separate `dbCreated` flag.